### PR TITLE
Topic/fix non default server

### PR DESCRIPTION
**Pull request by LFSaw**
+ use synth as target rather than its nodeID
+ believe in successful Node registration, needed for slow network connections.

### CODE REVIEW

File: Classes/Steno.sc [hunk 1]
```diff
@@ -216,10 +216,14 @@ Steno {
 			if(names.isNil or: { names.includes(token) }) {
 				args = argList.at(i);
 				oldSynth = oldSynthList.at(i);
-				newSynth = this.newSynth(token, nil, args, oldSynth.nodeID);
+
+				server.openBundle;
```
Comment (telephon) [inline Classes/Steno.sc]: why is this needed? In the method `eval`, a bundle is opened anyhow already. I tend to use the outermost context for organizing bundling, just to keep the code simple.

**Review by telephon — CHANGES_REQUESTED** [review]

File: Classes/Steno.sc (hunk 1 above)
Comment (LFSaw) [reply Classes/Steno.sc]: removed bundling. (It is difficult to understand what is the "outermost context" from just reading the code. I wanted to be sure this happens in exactly that sequence because here it is important).
